src/Sulu/Bundle/MediaBundle/Tests/Unit/Media/ImageConverter/Transformations/CropTransformationTest.php

```diff
@@ -19,7 +19,7 @@ class CropTransformationTest extends AbstractTransformation
 {
     protected $transformationServiceName = 'sulu_media.image.transformation.crop';
 
-    protected function getDataList()
+    protected function getDataList(): array
```

Comment:
test method changes should be done on 2.5 branch if possible and non breaking changes